<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(188, 224, 245);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(248, 163, 163);</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(248, 163, 163);</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flag_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(248, 163, 163);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="gameEndingText">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(188, 224, 245);</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="seedNumberBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>45</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(248, 163, 163);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seed number:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="minTimerLCD">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(248, 163, 163);</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="secTimerLCD">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(248, 163, 163);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>sec</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
